components/crud-web-apps/jupyter/frontend/src/app/pages/form/form-default/form-image/form-image.component.html

```diff
@@ -4,8 +4,23 @@
   ML packages"
   icon="fa:fab:docker"
 >
+  <section>
```

Comment:
You don't need this wrapper element. The `<mat-button-toggle-group>` will suffice

Reply:
Looked at it again and in conjunction with how we should place the `Custom image` checkbox.

I propose to 
1. use a wrapper `div` around the `<mat-button-toggle-group>` and the checkbox
2. that `div` will have the class `flex-column` which is defined in the common ui lib
3. move the checkbox on top of the buttons group

The end result will look like this 
![Screenshot from 2021-03-16 22-01-49](https://user-images.githubusercontent.com/11134742/111372400-43c97700-86a3-11eb-99a8-68ec1ac6d3e9.png)